crt_portal/cts_forms/templates/forms/complaint_view/index/filters/section_filter.html

```diff
@@ -1,14 +1,14 @@
 {% extends "forms/complaint_view/index/filter.html" %}
```

Comment:
Ins't this file part of Haseeb's work. May run into merge conflicts, check with him please.